README.md

```diff
@@ -80,6 +80,10 @@ We've provided some very, very basic recipes and are interested in adding more,
 so please send a pull request if you think anything would serve the larger
 community.
 
+## More information
```

Comment:
@dgrnbrg I was thinking of this as the section that would eventually contain links to various other README's in the repo - for https://github.com/twosigma/satellite/issues/43 .   How about "Subproject summary" a la cook?